docs/plugindev/README.md

```diff
@@ -232,6 +232,34 @@ make inventory-init
 make inventory-plugin-add
 ```
 
+### Updating the plugin group
+
+Create `plugin-scope-association.yaml` under `$(ROOT_DIR)/cmd/plugins/plugin-scope-association.yaml` of plugin repository
+
+Sample file content
+
+```yaml
+plugins:
+- name: builder
+  target: global
+  isContextScoped: false
+- name: test
+  target: global
+  isContextScoped: false
+```
+
+Update the PLUGIN_SCOPE_ASSOCIATION_FILE in plugin-tooling.mk to file location
+
+``` makefile
+PLUGIN_SCOPE_ASSOCIATION_FILE ?= $(ROOT_DIR)/cmd/plugins/plugin-scope-association.yaml
+```
+
+Run the below command to add plugin group with the respective plugins
+
```

Comment:
Can you specify that the `name:version` should be chosen?  
We need to explain the name and version format.

Reply:
added more details on format with examples